src/main/java/io/confluent/connect/hdfs/tools/SchemaSourceTask.java

```diff
@@ -62,6 +71,25 @@
       .field("id", Schema.INT32_SCHEMA)
       .field("seqno", Schema.INT64_SCHEMA)
       .build();
+
+
+  private static Schema valueSchemaLogical = SchemaBuilder.struct().version(1).name("record")
```

Comment:
let's give those variables names other than foobar1, foobar2. What's the difference between them?